components/kubearchive/policies/bootstrap-namespace.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: init-ns-kubearchiveconfig
+spec:
+  rules:
+  - name: init-ns-kubearchiveconfig
+    match:
+      any:
+      - resources:
+          kinds:
+          - Namespace
+          selector:
+            matchExpressions:
+            - key: toolchain.dev.openshift.com/type
+              operator: In
+              values:
+              - tenant
+      - resources:
+          kinds:
+          - Namespace
+          selector:
+            matchExpressions:
+            - key: konflux.ci/type
+              operator: In
+              values:
+              - user
+    generate:
+      generateExisting: true
```

Comment:
I'm not familiar with Kyverno. Will this be applied on top of the existing an existing, making their `spec.resources` empty? Say I initialized my namespace and I modified the KubeArchiveConfig. Will this rule make it empty again at some point?

Reply:
cf. https://kyverno.io/docs/writing-policies/generate/#generate-for-existing-resources

Reply:
as #5759  was merged, I added some e2e tests that are testing this scenario. PTAL https://github.com/redhat-appstudio/infra-deployments/blob/401da7865508f1e05f898fed85505b2ad1a7172e/components/kubearchive/policies/.chainsaw-test/chainsaw-test.yaml#L160-L318